test/functional/interface_rest.py

```diff
@@ -211,28 +213,30 @@ def run_test(self):
         self.test_rest_request('/block/{}'.format(bb_hash))
         self.nodes[0].reconsiderblock(bb_hash)
 
+        BLOCK_HEADER_SIZE = len(CBlockHeader().serialize())
```

Comment:
As this constant is used in multiple places, wouldn't it be an idea to define it in a shared place?

Reply:
(and added an assert right after for the invariant of 80)